common-lib/src/main/java/com/google/gct/idea/util/PlatformInfo.java

```diff
@@ -5,10 +5,16 @@
 
 import org.jetbrains.annotations.NotNull;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class PlatformInfo {
```

Comment:
I just rewrote to save system props now. For some reason I thought tests didn't preserve state, but obviously I'm wrong